src/Processors/AugmentTags.php

```diff
@@ -15,6 +15,22 @@
  */
 class AugmentTags implements ProcessorInterface
 {
+
+    /** @var array<string>  */
+    protected $unusedTagsToKeepWhitelist = [];
+
+    public function __construct(array $unusedTagsToKeepWhitelist = [])
```

Comment:
```suggestion
    public function __construct(array $whitelist = [])
```

It think this should be meaningful enough? This class is all about tags, after all.